src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -366,6 +378,22 @@ object SnowflakeTelemetry {
     }
     metric
   }
+
+  private[snowflake] def detectSparkLanguage(sparkConf: SparkConf): String = {
+    if (sparkConf.contains("spark.r.command")
+      || sparkConf.contains("spark.r.driver.command")
+      || sparkConf.contains("spark.r.shell.command")) {
+      "R"
+    } else if (sparkConf.contains("spark.pyspark.python")) {
+      "Python"
```

Comment:
@sfc-gh-ema `sparkConf.get("spark.pyspark.python").split("/").last` will get the binary name. It may include the version number, but it is not reliable. Since we can't detect the exact version, I  think we should use "Python"?